CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.5.0 (2020-11-11)
+
+- Added a phone field field testing fixture. [#77](https://github.com/blackbaud/skyux-phone-field/pull/77)
```

Comment:
Got an extra "field" in there. Otherwise, looks good :)

Reply:
aww dammit